Octokit/Models/Response/CommitCommentReaction.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Octokit.Internal;
+using System;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    public enum Reaction
+    {
+        [Parameter(Value = "plus_one")]
```

Comment:
The [api doco](https://developer.github.com/v3/reactions/#reaction-types) seems to indicate the string is "+1" and "-1" for these, but you've got their API string value as `plus_one` and `minus_one`.  Just checking if this was intentional?